doc/compiler/runtime/J9JITExceptionTable.md

```diff
@@ -0,0 +1,437 @@
+<!--
+Copyright IBM Corp. and others 2023
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] https://openjdk.org/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# Background
+
+The `J9JITExceptionTable` (also `typedef`'d to `TR_MethodMetaData`)
+is generally referred to as "the metadata". It contains most of the
+metadata associated with a compiled body[^1]. Most of this metadata
+is stored in the variable length section of the `J9JITExceptionTable`
+structure. This document outlines this variable length section. It does
+not describe fields that is described by a data structure (e.g., the
+`J9JITExceptionTable`, `TR_InlinedCallSite`, etc.).
+
+# Structure
+
+## High Level Overview
+The following diagram provides the high level overview. The
+`J9JITExceptionTable` is allocated and populated in
+[`createMethodMetaData`](https://github.com/eclipse-openj9/openj9/blob/v0.41.0-release/runtime/compiler/runtime/MetaData.cpp#L1315).
```

Comment:
It doesn't have to be a release branch, but ideally it would be at least a specific commit. The reason is because if I only use the main branch then if that function moves to a different line, the link will no longer be valid. So I can change it to a specific commit if you think that's better.

Reply:
Right.  The permalink does create it based on a particular commit:

https://github.com/eclipse-openj9/openj9/blob/dc712551de24413b6df6f64813216329aa0cd313/runtime/compiler/runtime/MetaData.cpp#L1315C8-L1315C8